evm/utils/test_builder/builder_utils.py

```diff
@@ -0,0 +1,126 @@
+import copy
+import itertools
+import random
+import subprocess
+
+from collections.abc import (
+    Mapping,
+)
+
+from evm.db.backends.memory import MemoryDB
+
+from cytoolz import (
+    first,
+    merge_with,
+)
+from eth_utils import (
+    force_text,
```

Comment:
`force_text` shouldn't be used.  It's been/being deprecated and removed.  If you need to coerce things from `bytes -> text` you should use `eth_utils.to_text` in combination with `eth_utils.text_if_str`

```python
from eth_utils.curried import text_if_str
from eth_utils import to_text

force_text = text_if_str(to_text)
```

The behavior isn't quite identical, but I think it's identical in all the ways that matter.  Note, that it **will not** be able to handle any arbitrary bytestring since not all 0-255 byte values are `utf8` encodable.

Reply:
Or, if you know the input will always be a `bytes` value, then `to_text(bytes_val)` is sufficient.